src/Nethermind/Nethermind.Consensus.AuRa/IActivatedAt.cs

```diff
@@ -22,16 +22,20 @@
 
 namespace Nethermind.Consensus.AuRa
 {
-    public interface IActivatedAtBlock
+    public interface IActivatedAt<out T> where T : IComparable<T>
```

Comment:
ah it can be block or time now - is it the case where it would be better to create two separate interface and we tried to find common meaning too hard?

Reply:
still just accept it as discussion and do not change it because of the review - it is ok and smart

Reply:
Yep generalization